spec/services/calculators/pensioner_capital_disregard_calculator_spec.rb

```diff
@@ -58,6 +58,16 @@ module Calculators
               let(:disposable_income) { 76.0 }
 
               it { is_expected.to eq 70_000.0 }
+
+              context "with 'pensioner_capital_disregard.minimum_age_in_years' threshold set to 'state_pension_age'" do
```

Comment:
I'm not sure about this test (and the next one) - its not clear to me what we are trying to assert (the answer is the same as the one above)